src/main/java/org/jenkinsci/test/acceptance/po/GlobalSecurityConfig.java

```diff
@@ -86,4 +86,13 @@ private void selectMarkupFormatter(final String formatter) {
 
         return newInstance(type, this, radio.getAttribute("path"));
     }
+
+    /**
+     * Decides whether script security for Job DSL scripts is enabled.
+     *
+     * @param enable Use script security if true.
+     */
+    public void setJobDslScriptSecurity(boolean enable) {
```

Comment:
Is this control only available for the job-dsl plug-in? If not remove job-dsl from the name of the method.